src/main/java/org/gridsuite/study/server/dto/impacts/AbstractBaseImpact.java

```diff
@@ -0,0 +1,51 @@
+/**
+  Copyright (c) 2023, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto.impacts;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.powsybl.iidm.network.IdentifiableType;
+
+import lombok.*;
```

Comment:
Avoid wildcard on imports

Reply:
Why ? It's everywhere in the code, even checkstyle or intelliJ auto replace multiple imports to this kind of wildcard imports from 5 or something like that

Reply:
I don't like it personnaly but you're right this is not necessarly bad.